tools/cldr-code/src/main/java/org/unicode/cldr/util/personname/PersonNameFormatter.java

```diff
@@ -400,6 +446,25 @@ public ParameterMatcher(Set<Length> lengths, Set<Style> styles, Set<Usage> usage
             this.usages = usages == null ? ImmutableSet.of() : ImmutableSet.copyOf(usages);
             this.orders = orders == null ? ImmutableSet.of() : ImmutableSet.copyOf(orders);
         }
+
+        public ParameterMatcher(String lengths, String styles, String usages, String orders) {
+            this.lengths = setFrom(lengths, Length::from);
+            this.styles = setFrom(styles, Style::valueOf);
+            this.usages = setFrom(usages, Usage::valueOf);
+            this.orders = setFrom(orders, Order::valueOf);
+        }
+
+        private <T> Set<T> setFrom(String parameter, Function<String, T> func) {
+            if (parameter == null || parameter.isBlank()) {
```

Comment:
@macchiati Maybe not the first, but first I've seen of JDK11 APIs in CLDR…  [isBlank()](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank())